base/regex.jl

```diff
@@ -303,8 +303,12 @@ struct SubstitutionString{T<:AbstractString} <: AbstractString
     string::T
 end
 
-endof(s::SubstitutionString) = endof(s.string)
-next(s::SubstitutionString, idx::Int) = next(s.string, idx)
+ncodeunits(s::SubstitutionString) = ncodeunits(s.string)
+codeunit(s::SubstitutionString) = codeunit(s.string)
+codeuint(s::SubstitutionString, i::Integer) = codeunit(s.string, i)
```

Comment:
Should this be `codeunit` rather than `codeuint`?

Reply:
why yes, yes it should. thank you!